es_manager.py

```diff
@@ -1,6 +1,7 @@
 import io
 import json
 import logging
+import sys
```

Comment:
This should be moved to after `pprint`: https://github.com/osu-mist/style-guides/blob/master/Python.md